subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -592,4 +596,78 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
         expect:
         succeeds("mytest", "assertNoTestClasses")
     }
+
+    def "multiple getTestingFramework() calls on a test suite return same instance"() {
```

Comment:
We could possibly delete these tests, but I like that we have something that will make the failure obvious if we ever revert by accident to the old behavior of new testingFramework instance per call.